addons/main/XEH_preInit.sqf

```diff
@@ -1,2 +1,13 @@
 #include "script_component.hpp"
-#include "XEH_PREP.hpp"
\ No newline at end of file
+#include "XEH_PREP.hpp"
+
+[
+	"AE3_DebugMode", // Settings internal name
+	"CHECKBOX", // Settings type
+	["Debug Mode", "By enabling the AE3 Debug Mode you will get additional information about the internal structure of AE3."], // Settings Name + Tooltip
+	"AE3 main", // Settings category
+	false, // Default Value
+    nil, // "_isGlobal" flag. Set this to true to always have this setting synchronized between all clients in multiplayer
```

Comment:
Should always be forced by the server settings

Reply:
Every CBA setting could be overwritten by mission or by server. There is no need to define that behavior at this point. If you think about using the `_isGlobal` flag, I think thats something different. This would sync the setting between all machines. We can't allow that because this would change the behavior for other players in a live mission as well.